sbd.spec

```diff
@@ -97,6 +105,17 @@ rm -rf %{buildroot}
 %postun
 %systemd_postun sbd.service
 %systemd_postun sbd_remote.service
+%else
+/sbin/chkconfig --add sbd_helper || :
+
+%preun
+/sbin/service sbd_helper stop  &>/dev/null || :
+/sbin/service sbd_remote_helper stop  &>/dev/null || :
+if [ $1 -eq 0 ]; then
+    # Package removal, not upgrade
+    /sbin/chkconfig --del sbd_helper || :
+    /sbin/chkconfig --del sbd_remote_helper || :
+fi
```

Comment:
I think I'd find the spec easier to follow if the %if were under %post, %preun etc. rather than one big %if for all of them.